SecuritySystemUWP/SecuritySystemUWP/Storage/OneDrive.cs

```diff
@@ -78,7 +75,6 @@ public DateTime LastUploadTime
                             var events = new Dictionary<string, string> { { "OneDrive", ex.Message } };
                             TelemetryHelper.TrackEvent("FailedToUploadPicture", events);
```

Comment:
Please also add TelemetryHelper.TrackException(ex), so we can get full data of the exception as well.